app/bundles/LeadBundle/Security/Permissions/LeadPermissions.php

```diff
@@ -32,7 +32,7 @@ public function __construct($params)
         $this->addExtendedPermissions('leads', false);
         $this->addExtendedPermissions('lists', false);
         $this->addStandardPermissions('imports');
-        $this->addCustomPermission('export', ['enable' => 1024]);
+        $this->addCustomPermission('export', ['enable' => 1024, 'notAnonymize' => 2]);
```

Comment:
Can we define constants for these values: 1024, 2?

Reply:
I followed the convention that is already in effect, this is how all permissions are granted in Mautic